source/Camera/Basic/Capture.hdev

```diff
@@ -75,4 +79,30 @@
 <parameters/>
 </docu>
 </procedure>
+<procedure name="get_first_available_zivid_device">
+<interface>
+<ic>
+<par name="ValueList" base_type="ctrl" dimension="0"/>
+</ic>
+<oc>
+<par name="Device" base_type="ctrl" dimension="0"/>
+</oc>
+</interface>
+<body>
+<l>for i := 0 to |ValueList| by 1</l>
+<l>    SplitValues := split(split(ValueList[i], '|'), ':')</l>
+<l>    if(SplitValues[2]{0:4} == 'Zivid')</l>
+<l>        Device := SplitValues[2]{0:strlen(SplitValues[2])-2}</l>
+<l>        break</l>
+<l>    endif</l>
+<l>endfor</l>
+<l>return ()</l>
+</body>
+<docu id="get_first_available_zivid_device">
+<parameters>
+<parameter id="Device"/>
+<parameter id="ValueList"/>
+</parameters>
+</docu>
+</procedure>
```

Comment:
Could you somehow move this function a shared file so all samples can use it?

Reply:
Good idea. I have no idea, I will try to figure out.

Reply:
Done!

Reply:
Looks good